### PR TITLE
Fuse support many routes

### PR DESCRIPTION
**Pull request by AndrewSC208**
Adds support for multiple processes to register routes with `fuse` and make them external on the control plane. This also adds debug settings for the `echo` process and updates the `echo` process to register with `fuse` as a secondary process for testing.

### CODE REVIEW

File: services/core/fuse/control_plane/controller.go [hunk 1]
```diff
@@ -81,17 +97,128 @@ func NewControlPlane(logger chassis.Logger) *controlPlane {
...
+		cp.logger.Error(err.Error())
+		return ErrUnableToSaveRoute
+	}
+
+	setReq := connect.NewRequest(&kvv1.SetRequest{
+		Key:   route.Name,
+		Value: val,
+	})
+
+	_, err = client.Set(context.Background(), setReq)
```
[the remark this answers is not in the document]
Comment (AndrewSC208) [reply services/core/fuse/control_plane/controller.go:116]: Good call. Updated.

File: services/core/fuse/control_plane/controller.go [hunk 2]
```diff
@@ -81,17 +97,128 @@ func NewControlPlane(logger chassis.Logger) *controlPlane {
...
+		Value: routeModel,
+	})
+
+	routes, err := client.List(context.Background(), listRoutesReq)
+	if err != nil {
+		cp.logger.Error(err.Error())
+		return ErrUnableToSaveRoute
+	}
+
+	fmt.Println(routes.Msg.GetValues())
```
[the remark this answers is not in the document]
Comment (AndrewSC208) [reply services/core/fuse/control_plane/controller.go:138]: updated

File: services/core/fuse/control_plane/snapshot.go [hunk 3]
```diff
@@ -110,79 +107,79 @@ func makeRoute(r *ntv1.Route) *route.RouteConfiguration {
...
-			// TransportSocket: &core.TransportSocket{
-			// 	Name: "envoy.transport_sockets.tls",
-			// 	ConfigType: &core.TransportSocket_TypedConfig{
-			// 		TypedConfig: tlspb,
-			// 	},
-			// },
-		}},
-	}
-}
+// func (cp *controlPlane) makeHTTPListener(listenerName string, r *ntv1.Route) *listener.Listener {
```
[the remark this answers is not in the document]
Comment (AndrewSC208) [reply services/core/fuse/control_plane/snapshot.go:110]: cleaned up

File: services/examples/echo/config.yaml [hunk 4]
```diff
@@ -1,3 +1,35 @@
...
+    # the bind address when opening ports for the web server
+    # default: "0.0.0.0"
+    bind_address: "localhost"
+    # the port the process will bind to (if it hosts a web server of any kind)
+    # default: 8090
+    port: 9090
+    # routing information within the draft cluster
+    internal:
+      # e.g. this could be http://file_host.examples.svc.cluster.local:8080 if running in kubernetes
+      host: 10.0.0.108
```
[the remark this answers is not in the document]
Comment (AndrewSC208) [reply services/examples/echo/config.yaml:29]: fixed

File: services/core/fuse/config.yaml [hunk 5]
```diff
@@ -6,7 +6,7 @@ service:
     port: 18000
 
 fuse:
-  address: http://localhost:18000
+  address: http://10.0.0.108:18000
```
[the remark this answers is not in the document]
Comment (AndrewSC208) [reply services/core/fuse/config.yaml]: updated

File: services/examples/file_host/config.yaml [hunk 6]
```diff
@@ -26,7 +26,7 @@ service:
     # routing information within the draft cluster
     internal:
       # e.g. this could be http://file_host.examples.svc.cluster.local:8080 if running in kubernetes
-      host: localhost
+      host: 10.0.0.108
```
Comment (jgkawell) [inline services/examples/file_host/config.yaml:29]: localhost

File: tests/fuse/config.yaml [hunk 7]
```diff
@@ -6,7 +6,7 @@ service:
     port: 18000
 
 fuse:
-  address: http://localhost:18000
+  address: http://10.0.0.108:18000
```
Comment (jgkawell) [inline tests/fuse/config.yaml:9]: localhost

File: services/examples/file_host/config.yaml [hunk 8]
```diff
@@ -19,17 +19,19 @@ service:
   network:
     # the bind address when opening ports for the web server
     # default: "0.0.0.0"
-    bind_address: "localhost"
+    bind_address: "0.0.0.0"
```
Comment (jgkawell) [inline services/examples/file_host/config.yaml:22]: probably should leave this as localhost

**Review by jgkawell — APPROVED** [review]

**Review by jgkawell — APPROVED** [review]

**Pull request merged**